deploy.ci.acceptance.mariadb.robot.yml

```diff
@@ -19,15 +19,14 @@ image:
         SPRYKER_PUSH_NOTIFICATION_WEB_PUSH_PHP_VAPID_PRIVATE_KEY: 'UK6DywwjKITPpRHBSY9TLPIXm6BPrHX40sseIoXT9c8'
         SPRYKER_PUSH_NOTIFICATION_WEB_PUSH_PHP_VAPID_SUBJECT: 'https://spryker.com'
         SPRYKER_YVES_HOST_EU: yves.eu.spryker.local
-        SPRYKER_YVES_HOST_US: yves.us.spryker.local
         SPRYKER_DYNAMIC_STORE_MODE: true
     node:
         version: 18
         npm: 9
 
 composer:
     mode: '--no-dev --quiet -o -a'
-    autoload: --classmap-authoritative
+    autoload: '--classmap-authoritative -o'
```

Comment:
Again, both lines are using the same -o and -a. `--classmap-authoritative` is the long form of `-a`. 

Why is it like this? Can you replace the duplication or what is the reason to have it?

Reply:
performance

Reply:
something not very important about format. All these changes for performance.